internal/command/flags/urfave.go

```diff
@@ -0,0 +1,60 @@
+package flags
+
+import (
+	"github.com/urfave/cli/v2"
+
+	"github.com/weaveworks/flintlock/internal/config"
+	"github.com/weaveworks/flintlock/pkg/defaults"
+)
+
+type WithFlagsFunc func() []cli.Flag
+
+func CLIFlags(options ...WithFlagsFunc) []cli.Flag {
```

Comment:
I like this component builder pattern here because that way when/if we add more binaries (now exporter, maybe a helper to hide a lot of "setup" steps into a single binary to deploy, an agent that can sit in the microvm, or basically anything) we can reused the same flags. Maybe it will not work long term, but now it has a nice structure.